dht/testing.go

```
@@ -41,7 +44,11 @@ func TestingCreateDHT(numNodes int, bootstrap, concurrent bool) (*BootstrapNode,
 			panic(err)
 		}
 
-		go dht.Start()
+		go func() {
+			if err := dht.Start(); err != nil {
+				logrus.Error("error starting dht - ", err)
```

similarly, this should `t.Error()`. or better yet, it should return the error, so the test can handle it itself